corehq/apps/data_dictionary/models.py

```diff
@@ -8,7 +8,19 @@
     ('plain', 'Plain'),
     ('number', 'Number'),
     ('select', 'Select'),
-    ('integer', 'Integer'),
+    ('audio', 'Audio'),
+    ('barcode', 'Barcode'),
+    ('gps', 'GPS'),
+    ('image', 'Image'),
+    ('phone_number', 'Phone Number'),
+    ('password', 'Password'),
+    ('signature', 'Signature'),
+    ('video', 'Video'),
+    ('balance', 'Balance'),
+    ('transfer', 'Transfer'),
+    ('dispense', 'Dispense'),
+    ('receive', 'Receive'),
+    ('save_to_case', 'Save To Case'),
```

Comment:
Maybe I'm confused about why they have human-friendly names at all. No need to answer if this is irrelevant.